tests/DevBetterWeb.Tests/Services/DailyCheckPingServiceTests/InvitationBuilder.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using DevBetterWeb.Core.Entities;
+
+namespace DevBetterWeb.Tests.Services.DailyCheckPingServiceTests
+{
+  public static class InvitationBuilder
```

Comment:
Builders typically shouldn't be static. If you run your tests in parallel, the shared state of static builders will cause random errors. Make it non-static, and then each test class that needs it should have a private field that is the builder that is instantiated when the test is created:

`private InvitationBuilder _invitationBuilder = new();`

Then use `_invitationBuilder` anywhere you need it in your tests.